src/view/dataElements/instanceNameOnly.jsx

```diff
@@ -40,16 +35,11 @@ const getSettings = values => {
   };
 };
 
-const validationSchema = object().shape({
-  instanceName: string().required("Please specify an instance")
-});
-
```

Comment:
Notice that since we're auto-selecting an instance name now, the instance name field will always have an instance selected, which negates the need for any validation on the field.